src/api/api.js

```diff
@@ -70,6 +70,17 @@ export const filtersReducer = (acc, filter = {}) => ({
     ...'osFilter' in filter && { osFilter: filter.osFilter }
 });
 
+function processOrderBy(orderBy) {
+    //TODO: remove logging
+    console.log("porcessing orderBy:\n orderBy: "+orderBy);
+    if (orderBy === "system_profile") {
+        orderBy = "operating_system";
+    }
+
+
+    return orderBy;
+}
```

Comment:
I'm not a fan of this solution because it limits us down the line from being able to have more `system_profile` columns. This likely will be a problem down the line, but there are also other design choices that are limiting us in the same way and fixing them all would be out of scope for this PR (IMO).

If there are other low-cost solutions that are better I'd love to hear thoughts, this is still an unfamiliar code base.

Reply:
New version removes this.